templates/zerver/help/deactivate-or-reactivate-a-user.md

```diff
@@ -14,12 +14,28 @@ Note that organization administrators cannot deactivate organization owners.
 
 {start_tabs}
 
+{tab|via-user-profile}
+
+1. Hover over a user's name in the right sidebar.
+
+1. Click on the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>)
+   to the right of their name.
+
+1. Click **Manage this user**.
+
+1. Click the **Deactivate user** button at the bottom of the **Manage user**
```

Comment:
Let's just make this: "Click the **Deactivate user** button at the bottom." I think it's clear enough, and I don't really want to introduce the term "interface".

Reply:
wfm, changed.